src/features/github/services/config/github-config-mutator-service.ts

```diff
@@ -0,0 +1,71 @@
+import _ from "lodash";
+import { AbstractConfigService } from "../../../../classes/abstract-config-service";
+import { PartialNested } from "../../../../types/partial-nested";
+import { GithubConfigValueNameEnum } from "../../enums/github-config-value-name.enum";
+import { IGithubConfig } from "../../interfaces/github-config";
+import { GithubConfigCoreService } from "./github-config-core-service";
+import { GithubConfigService } from "./github-config-service";
+
+export class GithubConfigMutatorService extends AbstractConfigService<
+  IGithubConfig
+> {
+  private static _instance: GithubConfigMutatorService;
+
+  public static getInstance(
+    config?: Readonly<PartialNested<IGithubConfig>>
+  ): GithubConfigMutatorService {
+    if (_.isNil(GithubConfigMutatorService._instance)) {
+      GithubConfigMutatorService._instance = new GithubConfigMutatorService(
+        config
+      );
+    }
+
+    return GithubConfigMutatorService._instance;
+  }
+
+  private readonly _githubConfigCoreService = GithubConfigCoreService.getInstance();
+  private readonly _githubConfigService = GithubConfigService.getInstance();
+  protected readonly _className = `GithubConfigMutatorService`;
+
+  protected constructor(config?: Readonly<PartialNested<IGithubConfig>>) {
+    super(config);
+  }
+
+  // @todo add coverage
```

Comment:
why you need to add coverage after ? Can you do coverage with this feature ?

Reply:
Yes, a bit lazy that is all.
This do it then :(